shell/dialog/DeactivateDriverDialog.vue

```diff
@@ -31,15 +31,15 @@ export default {
   },
   computed: {
     formattedText() {
-      const namesSliced = this.drivers.map((obj) => obj.nameDisplay).slice(0, 5);
-      const remaining = this.drivers.length - namesSliced.length;
+      const namesSliced = this.drivers.map((obj) => obj.nameDisplay);
```

Comment:
nitpick: `namesSliced` is really a slice any more. I would update this variable name, but it's not really necessary.